src/config/dynamic-theme-fixes.config

```diff
@@ -26314,15 +26314,8 @@ IGNORE IMAGE ANALYSIS
 streamable.com
 
 INVERT
-.customer-logos
```

Comment:
This was used in https://github.com/darkreader/darkreader/pull/9866

And now I see it changed to .customer-logos-v2

Please update.